src/test/convert-to-single-host.ts

```diff
@@ -11,7 +11,7 @@ import { promisify } from "util";
 const hosts = ["excel", "onenote", "outlook", "powerpoint", "project", "word"];
 const manifestFile = "manifest.xml";
 const packageJsonFile = "package.json";
-const prerelease = process.argv.indexOf('--prerelease') > -1;
+const prerelease = process.argv.indexOf('--prerelease') > -1 || true;
```

Comment:
Please add the comment that the `|| true` is to force using prerelease templates for now, and will be removed once the templates have been released which support convert-to-single-host.